lib/src/main/java/com/orange/ods/compose/component/banner/OdsBanner.kt

```diff
@@ -191,29 +192,32 @@ object OdsBanner {
 
 @UiModePreviews.Default
 @Composable
-private fun PreviewOdsBanner(@PreviewParameter(OdsBannerPreviewParameterProvider::class) parameter: OdsBannerPreviewParameter) =
-    OdsPreview {
-        with(parameter) {
-            OdsBanner(
-                message = message,
-                firstButton = firstButtonText?.let { OdsBanner.Button(it) {} },
-                image = imageRes?.let { OdsBanner.Image(painterResource(id = it), "") },
-                secondButton = secondButtonText?.let { OdsBanner.Button(it) {} },
-            )
-        }
+internal fun PreviewOdsBanner(@PreviewParameter(OdsBannerPreviewParameterProvider::class) parameter: OdsBannerPreviewParameter) {
```

Comment:
If I understood well, this function is only needed to display the preview in AS and the other one is used by paparazzi.
This one can be `private` I think and same thing for `OdsTopAppBar`.